util/skycultures/po4a.config

```diff
@@ -1,6 +1,9 @@
 [po4a_paths] ../../po/stellarium-skycultures-descriptions/stellarium-skycultures-descriptions.pot
 [type: xhtml] ../../skycultures/al-sufi/description.en.utf8
 [type: xhtml] ../../skycultures/anutan/description.en.utf8
+[type: xhtml] ../../skycultures/arabian_peninsula/description.en.utf8
+[type: xhtml] ../../skycultures/arabic/description.en.utf8
+[type: xhtml] ../../skycultures/arabic_lunar_stations/description.en.utf8
 [type: xhtml] ../../skycultures/arabic_moon_stations/description.en.utf8
```

Comment:
The sky culture arabic_moon_stations is removed and it should be removed from po4a.config file also.

Reply:
Thanks Alex,
Done.